s3/src/main/scala/geotrellis/spark/io/s3/package.scala

```diff
@@ -7,5 +7,14 @@ package object s3 {
   def makePath(chunks: String*) =
     chunks.filter(_.nonEmpty).mkString("/")
 
-  implicit class withSaveToS3Methods[K](rdd: RDD[(K, Array[Byte])]) extends SaveToS3Methods[K](rdd)
+  implicit class withSaveToS3Methods[K](rdd: RDD[(K, Array[Byte])]) {
```

Comment:
Good catch, wasn't thinking about that. Not sure the `Implicits` is appropriate here though. This is a separate project this has to be imported by name and nothing can extend it to bring it in.